src/renderer/components/views/Header.tsx

```diff
@@ -14,9 +15,7 @@ const panels = [
   ["images", "Images"],
 ];
 
-export default function Header({ panelSwitch }: Props) {
-  const [panelName, setPanelName]: [string, any] = panelSwitch;
-
+export default function Header({ panelName, setPanelName }: Props) {
```

Comment:
I've found that passing in the direct results of useStates as a tuple can be confusing/harder to follow than just passing in the individual values